lib/lrama/erb.rb

```diff
@@ -5,7 +5,7 @@
 
 module Lrama
   class ERB
-    # @rbs (String file, Hash[Symbol, untyped]) -> string
+    # @rbs (String file, **Hash[Symbol, untyped] kwargs) -> String
```

Comment:
I think `**untyped kwargs` is correct.

Reply:
See 1 of https://github.com/ruby/lrama/pull/586

Reply:
oh... thank you :bow: